source/fundamentals/enterprise-authentication.txt

```diff
@@ -2,4 +2,264 @@
 
 ====================================
 Enterprise Authentication Mechanisms
-====================================
\ No newline at end of file
+====================================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to authenticate with MongoDB using the
+**authentication mechanisms** available only in the {+mongo-enterprise+}.
+Authentication mechanisms are processes by which the driver and server confirm
+the identity of a client to ensure security before connecting.
+
+You can use the following authentication mechanisms with the latest version of
+{+mongo-enterprise+}.
```

Comment:
Issue: This sentence introduces the following links with "You can use the following authentication mechanisms...", but the two links start with "Authenticate with..." 

Suggestion: 
```suggestion
You can authenticate in the following ways with the latest version of
{+mongo-enterprise+}.
```
or:
```suggestion
You can authenticate against the latest version of
{+mongo-enterprise+} in the following ways:
```